### PR TITLE
Miscellaneous changes

### PR DESCRIPTION
**Pull request by wkaluza**

### CODE REVIEW

File: libs/vm_modules/include/vm_modules/ml/optimisation/adam_optimiser.hpp [hunk 1]
```diff
@@ -19,11 +19,11 @@
 
 #include "ml/graph.hpp"
 #include "ml/optimisation/adam_optimiser.hpp"
-
 #include "vm_modules/math/tensor.hpp"
```
Comment (baykaner) [inline libs/vm_modules/include/vm_modules/ml/optimisation/adam_optimiser.hpp]: what's going on here? seems to be only a change in whitespace?
is this done automatically by apply_style script? if so how can it be that it wasn't done previously?

**Review by baykaner — COMMENTED** [review]
looked over the math, ml, and vm_modules changes and all seems good. Better to get someone more familiar with it to review the other parts.

File: libs/vm_modules/include/vm_modules/ml/optimisation/adam_optimiser.hpp (hunk 1 above)
Comment (wkaluza) [reply libs/vm_modules/include/vm_modules/ml/optimisation/adam_optimiser.hpp]: This is a manual change - I removed a blank line.

File: libs/core/include/core/commandline/vt100.hpp [hunk 2]
```diff
@@ -25,56 +25,17 @@ namespace fetch {
 namespace commandline {
 namespace VT100 {
 
-uint16_t    ColorFromString(std::string name);
 std::string GetColor(int const &fg, int const &bg);
-std::string GetColor(std::string const &f, std::string const &b);
 
-inline static constexpr char const *DefaultAttributes()
+inline constexpr char const *DefaultAttributes()
```
Comment (ejfitzgerald) [inline libs/core/include/core/commandline/vt100.hpp]: Double check but I think that we can remove almost all of these VT100 files

File: libs/math/include/math/tensor.hpp [hunk 3]
```diff
@@ -210,7 +210,6 @@ class Tensor
 
   bool Resize(SizeVector const &shape, bool copy = false);
   bool Reshape(SizeVector const &shape);
-  bool ResizeFromShape(SizeVector const &shape);
```
Comment (ejfitzgerald) [inline libs/math/include/math/tensor.hpp:213]: @baykaner will probably have an opinion about removing this

**Review by ejfitzgerald — CHANGES_REQUESTED** [review]

**Comment by ejfitzgerald** [comment]
@wkaluza lets get @baykaner to review the tensor changes. I would simply remove all the vt100 stuff and then we should merge this

File: libs/math/include/math/tensor.hpp (hunk 3 above)
Comment (wkaluza) [reply libs/math/include/math/tensor.hpp:213]: @baykaner for the avoidance of doubt, this is a declaration without a definition - this function cannot be called atm.

File: libs/core/include/core/commandline/vt100.hpp (hunk 2 above)
Comment (wkaluza) [reply libs/core/include/core/commandline/vt100.hpp]: I had checked at the time. The fragments I left are used elsewhere, I think.

File: README.md [hunk 4]
```diff
@@ -42,7 +42,7 @@ cd ledger
 ./scripts/quickstart.sh
 ```
 
-Manual build steps are available in our [Getting Started Guide](docs/source/getting_started/index.rst).
+Or follow our online documentation at [building the ledger](http://etch-docs.economicagents.com/getting-started/installation-mac/).
```
Comment (ejfitzgerald) [inline README.md]: This link is incorrect should docs.fetch.ai

**Review by ejfitzgerald — APPROVED** [review]

**Pull request merged**